go/vt/vttablet/customrule/filecustomrule/filecustomrule.go

```diff
@@ -102,6 +108,47 @@ func ActivateFileCustomRules(qsc tabletserver.Controller) {
 	if *fileRulePath != "" {
 		qsc.RegisterQueryRuleSource(FileCustomRuleSource)
 		fileCustomRule.Open(qsc, *fileRulePath)
+
+		if !*fileRuleShouldWatch {
+			return
+		}
+
```

Comment:
nit: same change could be made for `fileRulePath` as was done for `fileRuleShouldWatch`